includes/hub/admin/class-event-log.php

```diff
@@ -8,14 +8,13 @@
 namespace Newspack_Network\Hub\Admin;
 
 use Newspack_Network\Admin as Network_Admin;
+use const Newspack_Network\constants\EVENT_LOG_PAGE_SLUG;
```

Comment:
This new constant needs to be imported in `class-users.php`, too: https://github.com/Automattic/newspack-network/blob/assorted-changes-2/includes/class-users.php#L78

Reply:
Fixec in 8f91df9d45cebd74ce4f9ea3082386cc0f99ed0f, thanks!